backend/src/controllers/posts.js

```diff
@@ -1,37 +1,40 @@
 const Post = require('../models/Post')
 
-async function findAll (ctx) {
-  // Fetch all Post’s from the database and return as payload
+const findAll = async (ctx) => {
+  console.log('--------------->  Lista los post!', ctx)
   const posts = await Post.find({})
-  // ctx.body = posts
-  ctx.body = { message: 'greetings from you local API :P' }
+  ctx.body = posts
 }
 
-async function create (ctx) {
+const create = async (ctx) => {
   // Create New Post from payload sent and save to database
-  // const newPost = new Post(ctx.request.body)
-  // const savedPost = await newPost.save()
-  ctx.body = { message: 'greetings from you local API :P' }
+  try {
+    const newPost = await new Post(ctx.request.body)
+    await newPost.save()
+    ctx.body = newPost.toJSON()
+  } catch (err) {
+    ctx.throw(409)
+  }
 }
 
-async function destroy (ctx) {
-  // Get id from url parameters and find Post in database
-  const id = ctx.params.id
-  const post = await Post.findById(id)
-
-  // Delete Post from database and return deleted object as reference
-  const deletedPost = await post.remove()
-  ctx.body = deletedPost
+const destroy = async (ctx) => {
+  try {
+    const id = ctx.params.id
+    const post = await Post.findByIdAndRemove(id)
+    console.log(post)
+    console.log(id)
+    const deletedPost = await post.findByIdAndRemove()
+    ctx.body = deletedPost
```

Comment:
use `ctx.status = 204` instead, see [http code protocol](https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete)